ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py

```diff
@@ -599,3 +599,43 @@ def test_paddle_roll(
         shifts=shift,
         axis=axis,
     )
+
+
+# unsqueeze
+@st.composite
+def _unsqueeze_helper(draw):
+    shape = draw(st.shared(helpers.get_shape(), key="value_shape"))
+    valid_axes = []
+    for index, _ in enumerate(shape):
+        valid_axes.append(index)
+        valid_axes.append(-index - 1)
+
+    valid_axes.append(index + 1)
+    valid_axes.append(-index - 2)
+
+    return draw(st.sampled_from(valid_axes))
+
+
+@handle_frontend_test(
+    fn_tree="paddle.unsqueeze",
+    dtype_param_and_indices=_unsqueeze_helper(),
+)
+def test_paddle_unsqueeze(
+    *,
+    dtype_and_x,
```

Comment:
You are using `_unsqueeze_helper()` in the decorator, and are not passing in the value in the function below.
You are using `dtype_and_x`which is not there in the decorator.

Reply:
Hi @zaeemansari70 , thanks for the feedback. Took me a while to understand the test fixtures and I was not able to run the tests the first time. It seems to be working now, let me know if there are any other issues.

```sh
root ➜ /workspaces/ivy (paddle_unsqueeze) $ pytest ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py::test_paddle_unsqueeze
Use Database in ReadOnly Mode with local caching !
=========================================================================================== test session starts ============================================================================================
platform linux -- Python 3.10.0, pytest-7.4.0, pluggy-1.2.0
backend(s): all
device: cpu
number of Hypothesis examples: 25
rootdir: /workspaces/ivy
plugins: anyio-3.7.1, metadata-3.0.0, hypothesis-6.82.0, json-report-1.5.0
collected 5 items                                                                                                                                                                                          

ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py .....  
```